foundation/foundation-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -36,6 +36,12 @@
 		<requiredProperty key="licenseName">
 			<defaultValue>closed-source-license</defaultValue>
 		</requiredProperty>
```

Comment:
@d-ryan-ashcraft Is there a specific default value we'd prefer as a placeholder, like a generic Nexus value?

Reply:
I think this is fine - we can't really assume what a user will be leveraging